dashboard/settings.py

```diff
@@ -35,6 +35,9 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'dashboard.socialaccount.providers.fxa',
+    'landing',
+
```

Comment:
Ah. This must be why so many projects make an `<app_name>/` directory inside their `templates/` directory. Hmm ... That's a tough call which convention is better ...